tasks-by-users/Rehan-Khan-Engineer/05-product-ui-card-hover-effect/style.css

```diff
@@ -0,0 +1,233 @@
+* {
```

Comment:
Reset Css to `box-sizing: border-box;`

Reply:
Do you know what is default Box-sizing property?

Reply:
content-box @Abbas4tech

Reply:
> 

Now if i change it it will effect elements dimensions, so let it be. from next time i will remember.